main.go

```diff
@@ -82,7 +82,7 @@ func initFlags() {
 
 	flag.Var(&autoscalingGroupPrefixes, "autoscalingGroupName", "An autoscalingGroup prefix for monitor")
 	flag.Var(&protectedFrameworks, "protectedFrameworks", "The mesos frameworks to wait for kill the node")
-
+	flag.Var(&protectedTasksLabels, "protectedTaskLabels", "The Tasks protected with labels(boolean) to wait for kill the node")
```

Comment:
I don't 100% like this message, let's see if we can find a better description for this